config/initializers/omniauth.rb

```diff
@@ -1,3 +1,5 @@
+OmniAuth.config.logger = Rails.logger
```

Comment:
Someone told me to add this on StackOverflow...

Reply:
![](https://media3.giphy.com/media/YRtLgsajXrz1FNJ6oy/giphy.gif?cid=5a38a5a2gzoafkv0gmiokb8t9gurg0olwrqiclzzrrjmhtod&rid=giphy.gif)

Reply:
I saw this in the docs too, but wasnt sure what it does. Any idea?

Reply:
I think it helps with debugging and returning OAuth errors to the rails log...but I'm not 100% sure.